src/Literal.ts

```diff
@@ -13,15 +13,14 @@ interface Literal<T extends LiteralBase = LiteralBase> extends Runtype.Common<T>
 	value: T
 }
 
-/**
- * Be aware of an Array of Symbols `[Symbol()]` which would throw "TypeError: Cannot convert a Symbol value to a string"
- */
-const literal = (value: unknown) =>
+const literal = (value: unknown): string =>
 	Array.isArray(value)
-		? globalThis.String(value.map(globalThis.String))
-		: typeof value === "bigint"
-			? globalThis.String(value) + "n"
-			: globalThis.String(value)
+		? globalThis.String(value.map(literal))
+		: value !== null && typeof value === "object"
+			? "[object Object]"
```

Comment:
This short circuits before getting to the `String(value)` that throws an error. I was initially thinking of checking `Object.getPrototypeOf(value) === null` but that may not handle exceptional scenarios that [extend null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends#extending_null).